src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/themes/AceThemes.java

```diff
@@ -249,6 +280,8 @@ public void onError(ServerError error)
    private ThemeServerOperations themeServerOperations_;
    private final EventBus events_;
    private final Provider<UserState> state_;
+   private final Provider<UserPrefs> prefs_;
    private final String linkId_ = "rstudio-acethemes-linkelement";
+   private final String fontId_ = "rstudio-fontelement";
```

Comment:
`fontId_` (and, actually, `linkId_`) could be local variables inside `applyTheme()`.